extensions/amp-story/0.1/share.js

```diff
@@ -103,7 +103,7 @@ function buildProviderParams(opt_params) {
   const attrs = dict();
 
   if (opt_params) {
-    Object.keys(opt_params || {}).forEach(field => {
+    Object.keys(opt_params).forEach(field => {
```

Comment:
This is an error, `undefined` (due to the `=` in the type def) cannot be passed to `Object.keys`.

Reply:
The `if (opt_params)` in the above line already prevents `undefined` from being passed to `Object.keys`.

Reply:
@jridgewell PTAL

Reply:
Ah, didn't see that.